keps/sig-windows/2802-identify-windows-pods-apiserver-admission/README.md

```diff
@@ -0,0 +1,393 @@
+
+<!--
+Describe the consequences on existing workloads (e.g., if this is a runtime
+feature, can it break the existing applications?).
+
+NOTE: Also set `disable-supported` to `true` or `false` in `kep.yaml`.
+-->
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+The admission plugins/Kubelet can act based on the OS field in pod spec if set by the end-user
+
+###### Are there any tests for feature enablement/disablement?
+Yes, unit and e2es tests for feature enabled, disabled
+
+
+### Rollout, Upgrade and Rollback Planning
+
+
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+It shouldn't impact already running workloads. This is an opt-in feature since users need to explicitly set the OS parameter in the Pod spec i.e .spec.os field. 
+If the feature is disabled the field is preserved if it was already set in the presisted pod object, otherwise it is silently dropped.
+
+
+###### What specific metrics should inform a rollback?
+N/A
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+Not yet tested.
+
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+None. This feature will be additive
+
+
+### Monitoring Requirements
+
+
+###### How can an operator determine if the feature is in use by workloads?
+
+By looking at the pod's spec. Some fields like Security Constraint Contexts may not
+be applied to the pod spec if the proper OS has been set.
+
+###### How can someone using this feature know that it is working for their instance?
+
+Windows Pods and Linux Pods with proper OS field set in the pod spec would cause pods
+to run properly on desired OS
+
+
+- [x] Events
+  - Event Reason: Kubelet will send pod admitted/denied event.
+- [ ] API .status
+  - Condition name: 
+  - Other field: 
+- [ ] Other (treat as last resort)
+  - Details:
+
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+All the pods which have OS field set in the pod spec should have OS specific constraints/defaults applied.
+
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+
+- [ ] Metrics
+  - Metric name:
+  - [Optional] Aggregation method:
+  - Components exposing the metric:
+- [x] Other (treat as last resort)
+  - Details: Linux Specific SCCs not available on Windows pod specs and vice-versa
+
+###### Are there any missing metrics that would be useful to have to improve observability of this feature?
+
+
+### Dependencies
+
+
+###### Does this feature depend on any specific services running in the cluster?
+None
```

Comment:
kube-apiserver and kubelet, right?

It's important to list these because a kubelet is often mismatched to a kube-apiserver and so the change won't fully function until kubelets are updated.